Rubberduck.Parsing/ComReflection/DeclarationsFromComProjectLoader.cs

```diff
@@ -0,0 +1,307 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices.ComTypes;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor;
+
+namespace Rubberduck.Parsing.ComReflection
+{
+    public class DeclarationsFromComProjectLoader : IDeclarationsFromComProjectLoader
+    {
+        public IReadOnlyCollection<Declaration> LoadDeclarations(ComProject type, string projectId = null)
+        {
+            var declarations = new List<Declaration>();
+
+            var projectName = new QualifiedModuleName(type.Name, type.Path, type.Name, projectId);
+            var project = new ProjectDeclaration(type, projectName);
+            declarations.Add(project);
+
+            foreach (var alias in type.Aliases.Select(item => new AliasDeclaration(item, project, projectName)))
+            {
+                declarations.Add(alias);
+            }
+
+            foreach (var module in type.Members)
+            {
+                var moduleIdentifier = module.Type == DeclarationType.Enumeration || module.Type == DeclarationType.UserDefinedType
+                    ? $"_{module.Name}"
+                    : module.Name;
```

Comment:
since you're just prepending the `_` here, wouldn't it be easier to have:

```
var moduleIdentifier = (moduleType == DeclarationType.Enumeration || module.Type == DeclarationType.UserDefinedType ? 
   "_" : "") + module.Name;
```

Reply:
I do not really think that that change would help readability.